batch/pom.xml

```diff
@@ -221,6 +221,13 @@
   </profiles>
 
   <dependencies>
+
+		<dependency>
+			<groupId>com.google.apis</groupId>
+			<artifactId>google-api-services-healthcare</artifactId>
+			<version>v1-rev20200515-1.30.9</version>
+		</dependency>
```

Comment:
Commented about this and an alternative solution [here](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/issues/194#issuecomment-898706496).

Reply:
Fixed in latest commit